### PR TITLE
Fill out more of the terminology section.

### PR DESCRIPTION
**Pull request by dj2**
This PR cleans up and extends the terminology section of the spec. The
terms have been sorted and some of the missing terms from the glossary
have been copied over.

***
<a href="https://pr-preview.s3.amazonaws.com/dj2/FedCM/pull/136.html" title="Last updated on Nov 5, 2021, 1:18 AM UTC (80540ca)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/FedCM/136/599ef7a...dj2:80540ca.html" title="Last updated on Nov 5, 2021, 1:18 AM UTC (80540ca)">Diff</a>

### CODE REVIEW

**Comment by dj2** [comment]
Ping

File: spec/index.bs [hunk 1]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
+: <dfn>account</dfn>
+:: TODO(goto): find existing definition.
 
-A <dfn>global identifier</dfn> is a string that identifies a particular [=user=]
-independent of which site they're visiting (e.g. email addresses and phone numbers).
-Users generally have relatively few global identifiers and can usually list and
-recognize them. A goal of anti-tracking policy is to prevent [=user identifiers=] from
-becoming [=global identifiers=].
+: <dfn>authentication</dfn>
+:: Process used by an [=Identity Provider=] to achieve sufficient confidence in
```
Comment (samuelgoto) [inline spec/index.bs:314]: Some (oauth, WebAuthn, I think?)  use the term "ceremony" to describe protocol-y processes.

Did you copy/paste this from somewhere or did you write this yourself?

File: spec/index.bs [hunk 2]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
+    the binding between the user and a presented identity.
+
+    Note that in some discussions and documentation, the term _authentication_ is
+    used to refer to the [=federated sign-in=] process. However, the user does not
+    authenticate to the [=RP=] during [=federated sign-in=]. The user
+    authenticates to the [=IDP=], which then provides a claim to the [=RP=]
+    asserting the user’s identity. The user does not prove their identity to the
+    [=RP=].
```
Comment (samuelgoto) [inline spec/index.bs:322]: If "the user does not prove their identity to the RP", what is it that the user does?

File: spec/index.bs [hunk 3]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
-becoming [=global identifiers=].
+: <dfn>authentication</dfn>
+:: Process used by an [=Identity Provider=] to achieve sufficient confidence in
+    the binding between the user and a presented identity.
+
+    Note that in some discussions and documentation, the term _authentication_ is
+    used to refer to the [=federated sign-in=] process. However, the user does not
+    authenticate to the [=RP=] during [=federated sign-in=]. The user
+    authenticates to the [=IDP=], which then provides a claim to the [=RP=]
+    asserting the user’s identity. The user does not prove their identity to the
```
Comment (samuelgoto) [inline spec/index.bs:321]: linki-fy [=identity=]? it is usually really overloaded.

File: spec/index.bs [hunk 4]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
+    [=RP=].
+
+    See also:
+    * [[OIDC-Connect-Core#Terminology]]
+    * [[OIDC-Connect-Core#Authentication]]
+    * [[SAML-Glossary]]
+
+: <dfn>directed identifier</dfn>
+:: A [=user identifier=] that that is unique for each site the user visits. A
```
Comment (samuelgoto) [inline spec/index.bs:330]: [=site=]

File: spec/index.bs [hunk 5]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
+    goal of anti-tracking policy is to promote [=user identifiers=] to become
```
Comment (samuelgoto) [inline spec/index.bs:331]: add a link to the threat model where this is further expanded?

File: spec/index.bs [hunk 6]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
+    [=directed identifiers=].
+
+: <dfn>first party</dfn>
```
Comment (samuelgoto) [inline spec/index.bs:334]: isn't there another spec that we could bring this term from?

File: spec/index.bs [hunk 7]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
-: <dfn noexport>Website</dfn>
-:: TODO(goto): find existing definitions.
+: <dfn>id token</dfn>
+:: TODO(goto): find existing definition.
 
 : <dfn>Identity Provider</dfn>
 : <dfn>IDP</dfn>
-:: TODO(goto): find existing definition.
+:: A service that has information about the user and can grant that information
+    to [=Relying Parties=].
```
Comment (samuelgoto) [inline spec/index.bs:361]: Does the pluralization here of "Parties" work in the definitions?

File: spec/index.bs [hunk 8]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
+    [high level vs low level](https://w3ctag.github.io/design-principles/#high-level-low-level)
 
-: <dfn>high-level API</dfn>
-:: A use case specific API, as opposed to a [=low-level API=]. See also [high level vs low level](https://w3ctag.github.io/design-principles/#high-level-low-level).
+: <dfn>out-of-band</dfn>
+:: Outside of the user agent's context.
 
-: <dfn>low-level API</dfn>
-:: A general purpose API, as opposed to a [=high-level API=]. See also [high level vs low level](https://w3ctag.github.io/design-principles/#high-level-low-level)
+: <dfn>party</dfn>
```
Comment (samuelgoto) [inline spec/index.bs:382]: Isn't this defined somewhere else that we could bring from?

File: spec/index.bs [hunk 9]
```diff
@@ -307,87 +307,129 @@ which the [=RP=] can use to authenticate the user.
...
 :: TODO(goto): find existing definition.
 
-: <dfn>out-of-band</dfn>
-:: Outside of the user agent's context.
+: <df>unsanctioned tracking</dfn>
+:: [[UNSANCTIONED-TRACKING]]
 
-: <dfn>joining</dfn>
-::
+: <dfn>user</dfn>
```
Comment (samuelgoto) [inline spec/index.bs:425]: isn't this also something that we can bring from another spec?

File: spec/index.bs (hunk 1 above)
Comment (dj2) [reply spec/index.bs:317]: Copied out of the glossary: https://github.com/WICG/FedCM/blob/main/explainer/glossary.md#authentication

File: spec/index.bs (hunk 3 above)
Comment (dj2) [reply spec/index.bs:324]: We don't define identity anywhere. I filed a [bug](https://github.com/WICG/FedCM/issues/137) to add it to the glossary as a followup.

File: spec/index.bs (hunk 2 above)
Comment (dj2) [reply spec/index.bs:325]: @kenrb I believe you wrote the glossary entry? Can you answer this?

File: spec/index.bs (hunk 4 above)
Comment (dj2) [reply spec/index.bs]: done.

File: spec/index.bs (hunk 5 above)
Comment (dj2) [reply spec/index.bs:334]: done

File: spec/index.bs (hunk 6 above)
Comment (dj2) [reply spec/index.bs:337]: This is existing content moved down from line 318. We can probably link from somewhere but would prefer to do it in a future CL.

File: spec/index.bs (hunk 7 above)
Comment (dj2) [reply spec/index.bs:364]: Had to test it, but yes it appears to.

File: spec/index.bs (hunk 8 above)
Comment (dj2) [reply spec/index.bs:385]: This was moved down from line 314, would prefer to remove in a followup.

File: spec/index.bs (hunk 9 above)
Comment (dj2) [reply spec/index.bs:428]: This was moved down from line 330, would prefer to remove in a followup

File: spec/index.bs (hunk 2 above)
Comment (samuelgoto) [reply spec/index.bs:325]: If this is just a move, we can resolve this in a separate CL.

**Comment by samuelgoto** [comment]
Seems like there is a conflict: resolve and I'll merge?

**Comment by dj2** [comment]
Done.

**Pull request merged**